aten/src/ATen/native/native_functions.yaml

```diff
@@ -3155,6 +3246,18 @@
   dispatch:
     CPU, CUDA: amin_out
 
+- func: mps_convolution(Tensor self, Tensor weight, Tensor? bias, int[] padding, int[] stride, int[] dilation, int groups) -> Tensor
```

Comment:
Same as for the transpose: no need for this to be here.

Reply:
created a issue + TODO: https://github.com/pytorch/pytorch/issues/77394